pcsx2/CDVD/IsoFS/IsoFS.cpp

```diff
@@ -49,8 +50,9 @@ IsoDirectory::IsoDirectory(SectorSource& r)
 	uint i = 16;
 
 	m_fstype = FStype_ISO9660;
-
-	while( !done )
+	
+	bool loadFromDisc = g_Conf->CdvdSource != CDVD_SourceType::NoDisc;
+	while( !done && loadFromDisc)
```

Comment:
Not sure what you mean by else. The code might suppress the message, but it doesn't make sense for the code to be in the ISO file system code at all.